andela/travel_tool_front
# #161866832 Change request modal field label

#### What does this PR do?
This PR changes the request modal input label from 'Your direct report' to 'Your Manager'

#### Description of Task to be completed?
- change input field label

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change into the project directory -` cd travel_tool_front`
- Pull and checkout into this branch - `ch-change-request-modal-label-161866832`
- Install project dependencies using `yarn install`
- Clone the [Travela backend](https://github.com/andela/travel_tool_back)
- Run database migrations using `yarn run db:rollmigrate`
- Check `.env.example` and set up your `.env` appropriately
- Start the Server and Frontend with `make start` and log into the application
- Click to create a new travel request
- The form should have 'Your Manager' not 'Your direct report'
- Editing a created request should also show a label 'Your Manager'

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#161866832](https://www.pivotaltracker.com/story/show/161866832)


#### Screenshots (if appropriate)
<img width="1227" alt="screen shot 2018-11-13 at 3 44 35 pm" src="https://user-images.githubusercontent.com/32720508/48420647-1f6bd180-e75b-11e8-8871-9689e30401f8.png">


#### Questions:
